src/components/mce-editor.tsx

```diff
@@ -0,0 +1,78 @@
+// "use client";
+
+import * as tinymceReact from "@tinymce/tinymce-react";
+import { Dispatch, SetStateAction, useState } from "react";
+import parse from "html-react-parser";
+
+interface INote {
+  _id: string;
+  title: string;
+  content: string;
+}
```

Comment:
I saw this `INote` type interface defined somewhere maybe in `page.tsx` why not create a file for all putting all your types and export/import them in the places where you did them to avoid duplications and makes it easier to maintain if you wanted to add more types properties to it, that way you'll only have to make the changes in one place.
WDYT?